src/api/assets/assets.md

```diff
@@ -66,9 +67,6 @@ Money assets have the following fields along with the base asset fields.
 | Field    | Type               | Description                                                          |
 |:---------|:-------------------|:---------------------------------------------------------------------|
 | currency | String             | Currency code, eg "NZD"                                              |
-| balance  | {% dt BigNumber %} | Current balance in the currency's smallest denomination (ie. cents). |
```

Comment:
Can we keep this field around?

Because we don't have EXPERIMENTAL on it, we need to keep the API stable :)